webapp/public/js/domjudge.js

```diff
@@ -435,10 +435,13 @@ function processAjaxResponse(jqXHR, data) {
         window.location = jqXHR.getResponseHeader('X-Login-Page');
     } else {
         var newCurrentContest = jqXHR.getResponseHeader('X-Current-Contest');
-        var dataCurrentContest = $('[data-current-contest]').data('current-contest').toString();
+        var dataCurrentContest = $('[data-current-contest]').data('current-contest');
+        var refreshStop = $('[data-ajax-refresh-stop]');
 
-        // If the contest ID changed from another tab, reload or whole page
-        if (dataCurrentContest !== undefined && newCurrentContest !== dataCurrentContest) {
+        // Reload the whole page if
+        // - we were signaled to stop refreshing, or
+        // - if the contest ID changed from another tab.
+        if ("0" in refreshStop || (dataCurrentContest !== undefined && newCurrentContest !== dataCurrentContest.toString())) {
```

Comment:
I still don't get this `0` here, when we are setting a `="1"`

Reply:
See above for explanation.